app/models/filter.rb

```diff
@@ -135,6 +141,21 @@ def expire_topbar_cache(sweeper)
     role.usergroups.each { |g| g.expire_topbar_cache(sweeper) }
   end
 
+  def disable_overriding!
+    self.override = false
+    self.save!
+  end
+
+  def inherit_taxonomies
+    inherit_taxonomies! unless self.override?
```

Comment:
What about calling it `enforce_inherited_taxonomies`. The reason why I would prefer the current approach, rather than putting the `unless` into validations is, once we add this somewhere else, it would be detected as duplicate code (and we can let Rubocop doing it's job). I think the conditionals on validations are mainly meant for standard Rails validations, where you have no other option. I think keeping the association when override is check makes sense, therefore I don't get it as argument for this. The rest I think can be fixed just by renaming the method.